app/controllers/post_controller.rb

```diff
@@ -1,4 +1,4 @@
-class PostController < ApplicationController
+class PostController < BasePostEditorController
   # GET post/list
   def list
     @posts = GithubService.get_all_posts(session[:access_token])
```

Comment:
So, question here. This method looks as though it's intended to grab all the current posts from GitHub. In the user stories, we don't technically include a user story for editing existing posts (although I agree it's useful!). That said, I think we can leave the endpoint here for future development. Let's just make sure that the front-end doesn't give this intended functionality away (yet).

Reply:
Yes, that controller method currently returning a list of posts to be displayed in post/list. I didn't know that we were not considering editing MVP. In the future I'll be sure that we won't have editing as MVP but I'll keep some of the code that I started for editing.

Reply:
Awesome, thanks! Again, I'm sorry for the confusion. I guess during your demos, I knew what information you were pulling but I always figured that it was more of an example for showing off the previewing than an actual feature. We'll have to make sure we're on the same page before huge swaths of development in the future.